app/assets/javascripts/discourse/tests/acceptance/sidebar-user-community-section-test.js

```diff
@@ -564,6 +564,26 @@ acceptance("Sidebar - Logged on user - Community Section", function (needs) {
     );
   });
 
+  test("my posts title changes when drafts are present", async function (assert) {
+    await visit("/");
+
+    assert.strictEqual(
+      query(".sidebar-section-link-my-posts").title,
+      "My recent topic activity",
+      "displays the default title when no drafts are present"
+    );
+
+    await publishToMessageBus(`/user-drafts/${loggedInUser().id}`, {
+      draft_count: 1,
+    });
+
+    assert.strictEqual(
+      query(".sidebar-section-link-my-posts").title,
+      "My unposted drafts",
```

Comment:
```suggestion
     I18n.t("sidebar.sections.community.links.my_posts.title_drafts"),
```

We can just use `I18n` directly instead of manually typing the description here. This way, changes to translation strings in the future does not result in the test breaking.